src/common/helpers/format.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { format } from 'date-fns';
```

Comment:
Fisierul ast ar trebui sters in the end right?

Reply:
l-am sters